README.md

```diff
@@ -51,11 +49,11 @@ Potete suggerire cambiamenti in vari modi:
 
 + chi fosse invece più familiare con GitHub, può spingersi ad utilizzare il flusso di lavoro tipico in GitHub per modifiche sostanziali, che sfocia tipicamente in una ***pull request***, ma non è essenziale e non lo esigiamo in nessun modo, gli *issues* vanno benissimo
 
-+ al limite, va bene anche **una semplice segnalazione via mail** (in effetti, misura *anti-spam*, l'indirizzo è accessibile in *GitHub* solo *dopo* aver effettuato il login), sapendo che inviare mail espone a *metadata recollection*, ma *noi* stiamo comunque dalla parte della legalità e della Costituzione (potrete comunque usare *GPG* per segnalarci punti deboli del manifesto)
++ al limite, va bene anche **una semplice segnalazione via mail** (in effetti, misura *anti-spam*, l'indirizzo è accessibile in *GitHub* solo *dopo* aver effettuato il login), sapendo che inviare mail espone a *metadata recollection*, ma *noi* stiamo comunque dalla parte della legalità e della Costituzione. Potrete comunque usare *GPG* per segnalarci punti deboli del manifesto
```

Comment:
(   GPG lo suggerivo tra parentesi come "pssst, i punti particolarmente deboli passateceli in segreto così non servono al nemico"  :eye:   )

Inoltre, qui stai toccando un problema più generale: a volte ho chiuso gli items delle liste con un punto e a volte no.
Se metti punto e maiuscola, chiama un punto alla fine.
Quando è possibile, evito per lasciare graficamente più pulite le liste, ma a volte l'item è così articolato che non si può evitare. Qui lo lascerei com'era e poi lo rivediamo nel contesto.

Reply:
Cancelli questa modifica dalla pull? (idem come sopra se ti faccio lavorare inutilmente me lo dici, ci metto 5 minuti, metà schermo questa discussione l'altra metà il file, copio le modifiche e push)